Library/Homebrew/utils.rb

```diff
@@ -109,7 +109,13 @@ def odeprecated(method, replacement = nil, disable: false, disable_on: nil, call
 
   if ARGV.homebrew_developer? || disable ||
      Homebrew.raise_deprecation_exceptions?
-    developer_message = message + "Or, even better, submit a PR to fix it!"
```

Comment:
How about only setting this if `tap` and `replacement` are both set?

Reply:
👍Happy to look at that.